src/alcotest-stdlib-ext/alcotest_stdlib_ext.ml

```diff
@@ -45,6 +45,15 @@ module List = struct
     in
     aux [] n l
 
+  let concat_map f l =
+    let rec aux f acc = function
+      | [] -> rev acc
+      | x :: l ->
+          let xs = f x in
+          (aux [@tailcall]) f (rev_append xs acc) l
+    in
+    aux f [] l
```

Comment:
Any reason you are passing `f` to `aux`?

Reply:
This is just me over-optimising out of habit. `ocamlopt` w/o flambda doesn't do lambda lifting, so if you don't string `f` through the calls to `aux` then you need to allocate a closure to store it (which is slower).

https://godbolt.org/z/4zxvszW7c

Reply:
Thanks :)